cvat/apps/engine/cache.py

```diff
@@ -91,7 +92,9 @@ def enqueue_create_chunk_job(
         with get_rq_lock_for_job(queue, rq_job_id, blocking_timeout=blocking_timeout):
             rq_job = queue.fetch_job(rq_job_id)
 
-            if not rq_job:
+            # Enqueue the job if the chunk was deleted but the RQ job still exists.
+            # This can happen in cases involving jobs with honeypots.
+            if not rq_job or rq_job.get_status(refresh=False) in {RQJobStatus.FINISHED, RQJobStatus.FAILED}:
```

Comment:
```suggestion
            if not rq_job or (
                # Enqueue the job if the chunk was deleted but the RQ job still exists.
                # This can happen in cases involving jobs with honeypots and
                # if the job wasn't collected by the requesting process for any reason.
                rq_job.get_status(refresh=False) in {RQJobStatus.FINISHED, RQJobStatus.FAILED, RQJobStatus.CANCELED}
            ):
```

Reply:
Updated